multy_core/src/ethereum/ethereum_transaction.h

```diff
@@ -63,11 +70,10 @@ class EthereumTransaction : public TransactionBase
     EthereumTransactionFeePtr m_fee;
     EthereumTransactionSourcePtr m_source;
     EthereumTransactionDestinationPtr m_destination;
-    EthereumTokenTransferDataPtr m_token_transfer_data;
+    EthereumTransactionDestinationPtr m_internal_destination;
```

Comment:
Please add a comment:
A destination used only to simplify transaction serialization, can be generated from token transfer info or just a receiver info (in case of plain Ether transfer).